### PR TITLE
feat: Add `NavLink` component

### PR DESCRIPTION
**Pull request by maciejopalinski**

### CODE REVIEW

File: src/components/header/Header.js [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
+      </a>
       <div class="navBar">
         <ul id="menu" class="menu">
-          <li>Movie</li>
-          <li>TV Series</li>
-          <li>Search</li>
-          <li>Favorite</li>
+          ${NavLink({ href: '#', title: 'Movie' })}
+          ${NavLink({ href: '#', title: 'TV Series' })}
+          ${NavLink({ href: '/search', title: 'Search' })}
```
Comment (Veryyapeee) [inline src/components/header/Header.js]: href should be taken from `routes` object

File: src/components/Nav/Link.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.navBar li a {
+    color: white;
```
Comment (Veryyapeee) [inline src/components/Nav/Link.scss]: it's just white, but it should be still in variables.scss

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

File: src/components/Nav/Link.scss (hunk 2 above)
Comment (Veryyapeee) [reply src/components/Nav/Link.scss]: Also we are making dependency in here, in component we should keep scope of a component, so styles shouldn't depend on the .navBar class which is in other component. styling wouldn't work if we would like to put this component somewhere else, just add a class to li tag

**Comment by Veryyapeee** [comment]
Also 1 more thing, no one noticed that actually, even me :D

We need to be able to somehow go back to homePage, so we should add redirection to homeScreen on logo click, can you add this in this PR as well?

**Comment by Veryyapeee** [comment]
Also one last thing, in Base.js there is somehow missing import of FavoriteScreen, can you add that as well, it crashes the app :D

File: src/components/header/Header.js [hunk 3]
```diff
@@ -1,18 +1,19 @@
...
+import { NavLink } from '../Nav/Link';
 import { redirect } from '../..';
 import { routes } from '../../route';
 
-//TODO: connect li elements with redirect function
-
 const template = `
     <header>
-      <img src="/static/img/logo.png" alt="logo" class="logo">
+      <a href="/">
```
Comment (Veryyapeee) [inline src/components/header/Header.js]: routes object :P

Comment (maciejopalinski) [reply src/components/header/Header.js]: marudzisz ;)

**Review by Veryyapeee — CHANGES_REQUESTED** [review]
saerchbroke
SearchScreen
:o

**Review by Veryyapeee — APPROVED** [review]
lgtm 8)

**Review by Mariusz-web — APPROVED** [review]

**Pull request merged**